languages/python/bin/README.md

```diff
@@ -0,0 +1,24 @@
+# Miscellaneous Utilities
+
+These are various helper utilities to use in managing the Python track.
+
+## make-concept-exercise
+
+The purpose of this utility is to create the (somewhat complex) structure of a concept exercise. At the moment it's not particularly powerful, but it will populate SLUG/.meta/config.json for you, and will create SLUG/.meta/design.md with minimal structure.
+
+```
+$ ./bin/make-concept-exercise bool-basic --forked-from csharp/bool-basic
```

Comment:
The `-basic` suffix seems to imply some form of hierarchy. We've recently added [guidelines for naming concept](https://github.com/exercism/v3/blob/master/docs/maintainers/determining-concepts.md#naming-concepts) that include a "no hierarchical names" rule. Maybe you could use a different example as to not suggest using a hierarchy in their concept names?

Reply:
Fair enough, I just picked that one out of thin air, as I just want to demonstrate that the names can be hyphenated.

Reply:
Changed it to `boolean-logic`.